src/App.js

```diff
@@ -2,22 +2,32 @@ import React from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
-function App() {
+function useSemiPersistentState() {
+  const [todoList, setTodoList] = React.useState(
+    JSON.parse(localStorage.getItem("savedTodoList")) || React
```

Comment:
I cleared my local storage and your app broke. What should be on this line?

Reply:
instead of a string there should be an empty array? like this "JSON.parse(localStorage.getItem('savedTodoList'))|| []"

Reply:
Yes, an empty array.